src/keycloak/chart/templates/vs-emulate-gitlab.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Capabilities.APIVersions.Has "security.istio.io/v1beta1" }}
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: emulate-gitlab
+  namespace: {{ .Release.Namespace }}
```

Comment:
I'm sure I'm just missing the connecting dots, but why is this VS created via the Istio CR instead of the UDS Package? Match and rewrite should be supported in that spec and I think you dodge name conflicts using `description`?

Also not clear to me - does this need the same `istio-mtls-client-certificate` remove/add logic?

Reply:
This stuck out to me when setting up the flavors since I had to add the istio template conditional for the single package test and the reasoning wasn't apparent.

Reply:
I actually think it might have justs been a holdover--I added the advanced http stuff in this pr originally and then we spun it out